core/services/relay/evm/chain_writer.go

```diff
@@ -111,6 +111,11 @@ func (w *chainWriter) SubmitTransaction(ctx context.Context, contract, method st
 		checker.CheckerType = txmgrtypes.TransmitCheckerType(methodConfig.Checker)
 	}
 
+	v := big.NewInt(0)
```

Comment:
why is txmgr accepting big.Int without a pointer  😫

Reply:
We're busy discussing whether it's in scope to edit this too along with the other `big.Int` -> `*big.Int` changes we're making.

Reply:
But there is an argument that `nil` is not a valid value and using `big.Int` helps prevent accidental panics at the cost of being a little ugly.